go.mod

```diff
@@ -68,12 +71,12 @@ require (
 	github.com/subosito/gotenv v1.2.0 // indirect
 	github.com/urfave/cli v1.22.5 // indirect
 	go.uber.org/atomic v1.10.0 // indirect
-	go.uber.org/multierr v1.8.0 // indirect
-	golang.org/x/net v0.0.0-20220225172249-27dd8689420f // indirect
-	golang.org/x/sync v0.0.0-20220601150217-0de741cfad7f // indirect
-	golang.org/x/sys v0.0.0-20220520151302-bc2c85ada10a // indirect
-	golang.org/x/term v0.0.0-20210927222741-03fcf44c2211 // indirect
-	golang.org/x/text v0.3.7 // indirect
+	go.uber.org/multierr v1.9.0 // indirect
+	golang.org/x/net v0.3.0 // indirect
```

Comment:
This one is still outdated, unfortunately (CVE-2022-41723). Please, upgrade to 0.7.0+ in another commit.

Reply:
updated